From 151223bcd9a3e3707ff9bc77ce2812ebf0ee74a0 Mon Sep 17 00:00:00 2001
From: Hendrik Brueckner <brueckner@redhat.com>
Date: Thu, 10 Jul 2014 09:08:42 -0400
Subject: [s390] zcrypt: remove zcrypt kmsg documentation again

Message-id: <1404983322-13545-3-git-send-email-brueckner@redhat.com>
Patchwork-id: 85646
O-Subject: [RHEL7.1 PATCH 2/2] [s390] s390/zcrypt: remove zcrypt kmsg documentation again
Bugzilla: 1088511
RH-Acked-by: Don Zickus <dzickus@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

Description
-----------
commit 4dd841a0c4fe4d5be3825cf0e129d3da431d82b0
Author: Heiko Carstens <heiko.carstens@de.ibm.com>
Commit: Martin Schwidefsky <schwidefsky@de.ibm.com>

    s390/zcrypt: remove zcrypt kmsg documentation again

    This part of the ep11 patch should not have been merged.

    Signed-off-by: Heiko Carstens <heiko.carstens@de.ibm.com>
    Signed-off-by: Martin Schwidefsky <schwidefsky@de.ibm.com>

Bugzilla
--------
BZ 1088511
https://bugzilla.redhat.com/show_bug.cgi?id=1088511

Upstream status of the patch
----------------------------
The patch is upstream as of kernel version v3.14-rc1
http://git.kernel.org/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commitdiff;h=4dd841a0c4fe4d5be3825cf0e129d3da431d82b0

Brew
----
https://brewweb.devel.redhat.com/taskinfo?taskID=7667791

Test status
-----------
The patch has been tested and fixes the problem.
The fix has been verified by the IBM test department.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Documentation/kmsg/s390/zcrypt b/Documentation/kmsg/s390/zcrypt
deleted file mode 100644
index 7fb2087..0000000
--- a/Documentation/kmsg/s390/zcrypt
+++ /dev/null
@@ -1,20 +0,0 @@
-/*?
- * Text: "Cryptographic device %x failed and was set offline\n"
- * Severity: Error
- * Parameter:
- *   @1: device index
- * Description:
- * A cryptographic device failed to process a cryptographic request.
- * The cryptographic device driver could not correct the error and
- * set the device offline. The application that issued the
- * request received an indication that the request has failed.
- * User action:
- * Use the lszcrypt command to confirm that the cryptographic
- * hardware is still configured to your LPAR or z/VM guest virtual
- * machine. If the device is available to your Linux instance the
- * command output contains a line that begins with 'card<device index>',
- * where <device index> is the two-digit decimal number in the message text.
- * After ensuring that the device is available, use the chzcrypt command to
- * set it online again.
- * If the error persists, contact your support organization.
- */
-- 
1.7.1